src/OperationResults/OperationResults.csproj

```diff
@@ -18,13 +18,24 @@
         <RepositoryUrl>https://github.com/marcominerva/OperationResults.git</RepositoryUrl>
         <RepositoryBranch>master</RepositoryBranch>
         <PackageReadmeFile>README.md</PackageReadmeFile>
+        <PublishRepositoryUrl>true</PublishRepositoryUrl>
+        <EmbedUntrackedSources>true</EmbedUntrackedSources>
+        <DebugType>embedded</DebugType>
+    </PropertyGroup>
+
+    <PropertyGroup Condition="'$(GITHUB_ACTIONS)' == 'true'">
+        <ContinuousIntegrationBuild>true</ContinuousIntegrationBuild>
     </PropertyGroup>
 
     <ItemGroup>
         <PackageReference Include="Nerdbank.GitVersioning" Version="3.5.109">
             <PrivateAssets>all</PrivateAssets>
             <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
         </PackageReference>
+        <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1">
```

Comment:
NuGet references should be in alphabetic order.

Reply:
Fixed with 5f25a941423c9bed94670b0c42aad99702351d08